powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/ApplicationConfigController.java

```diff
@@ -91,10 +100,13 @@ public ObjectResponse<CreateApplicationConfigResponse> createApplicationConfig(@
      * @throws Exception In case the service throws exception.
      */
     @PostMapping("/remove")
-    public Response removeApplicationConfig(@RequestBody ObjectRequest<RemoveApplicationConfigRequest> request) throws Exception {
-        logger.info("RemoveApplicationConfig call received: {}", request);
-        service.removeApplicationConfig(request.getRequestObject());
-        logger.info("RemoveApplicationConfig succeeded.");
+    public Response removeApplicationConfig(@Valid @RequestBody ObjectRequest<RemoveApplicationConfigRequest> request) throws Exception {
+        final RemoveApplicationConfigRequest req = request.getRequestObject();
+        logger.info("action: removeApplicationConfig, state: initiated, applicationId: {}, key: {}", req.getApplicationId(), req.getKey());
+        logger.debug("action: removeApplicationConfig, state: initiated, request: {}", request);
+        service.removeApplicationConfig(req);
+        logger.info("action: removeApplicationConfig, state: succeeded");
+        logger.debug("action: removeApplicationConfig, state: succeeded, empty response");
```

Comment:
I suggest this change to make the parsing convenient even though there is no rich information to share. 
```suggestion
        logger.debug("action: removeApplicationConfig, state: succeeded, response: empty");
```